controllers/routingctrl/controller.go

```diff
@@ -20,10 +19,12 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
-const name = "routing"
+const (
+	name          = "routing"
+	finalizerName = "platform-routing.opendatahub.io/finalizer"
```

Comment:
To be consistent with annotations and labels:

```suggestion
	finalizerName = "routing.opendatahub.io/finalizer"
```